models/gpt2/gpt2.py

```diff
@@ -275,6 +275,25 @@ def get_activation_fn(params):
         return mtf.selu
     elif activation_fn == "elu": # https://arxiv.org/abs/1511.07289
         return mtf.elu
+
+    elif activation_fn == "abs": 
+        return mtf.abs
+    elif activation_fn == "id":
+        return lambda x: x
+    elif activation_fn == "sin":
+        return mtf.sin
+    elif activation_fn == "cos":
+        return mtf.cos
+    elif activation_fn == "sign":
+        return mtf.sign
+    elif activation_fn == "triangle relax":
+        return lambda x: mtf.sin(x)-mtf.sin(3*x)/9+mtf.sin(5*x)/25-mtf.sin(7*x)/49
+    elif activation_fn == "square relax":
+        return lambda x: mtf.cos(x)-mtf.cos(3*x)/3+mtf.cos(5*x)/5-mtf.cos(7*x)/7
```

Comment:
no spaces in names

Reply:
what about underscores, capitalization, or dashes? should the names really be `thisisaverylongname`?

Reply:
underscores are the preferred way, spaces are best avoided because they wreak all sorts of havoc.